install/installer/pkg/components/openfga/deployment.go

```diff
@@ -20,6 +23,122 @@ import (
 func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 	labels := common.CustomizeLabel(ctx, Component, common.TypeMetaDeployment)
 
+	cfg := getExperimentalOpenFGAConfig(ctx)
+	if cfg == nil || !cfg.Enabled {
+		return nil, nil
+	}
+
+	containers := []corev1.Container{
+		{
+			Name:            ContainerName,
+			Image:           ctx.ImageName(common.ThirdPartyContainerRepo(ctx.Config.Repository, RegistryRepo), RegistryImage, ImageTag),
+			ImagePullPolicy: corev1.PullIfNotPresent,
+			Args: []string{
+				"run",
+				"--log-format=json",
+				"--log-level=warn",
+			},
+			Env: common.CustomizeEnvvar(ctx, Component, common.MergeEnv(
+				common.DefaultEnv(&ctx.Config),
+			)),
+			Ports: []corev1.ContainerPort{
+				{
+					ContainerPort: ContainerGRPCPort,
+					Name:          ContainerGRPCName,
+					Protocol:      *common.TCPProtocol,
+				},
+				{
+					ContainerPort: ContainerHTTPPort,
+					Name:          ContainerHTTPName,
+					Protocol:      *common.TCPProtocol,
+				},
+				{
+					ContainerPort: ContainerPlaygroundPort,
+					Name:          ContainerPlaygroundName,
+					Protocol:      *common.TCPProtocol,
+				},
+			},
+			Resources: common.ResourceRequirements(ctx, Component, ContainerName, corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					"cpu":    resource.MustParse("1m"),
+					"memory": resource.MustParse("30Mi"),
+				},
```

Comment:
Where are these resource limits taken from, or are they an initial best guess?

Reply:
An initial one, just picked something small. They are just requests (not limits) so it will realistically consume more